crates/test-utils/src/fixtures.rs

```diff
@@ -32,6 +32,7 @@ pub async fn erc20(#[future] katana: Katana) -> (Katana, KakarotEvmContract) {
 
 /// This fixture creates a new test environment on Katana.
 #[fixture]
+#[allow(clippy::future_not_send)]
```

Comment:
Why do we need this

Reply:
see message is previous question

Reply:
please update this based on above comment